Makefile

```diff
@@ -37,7 +37,7 @@ docker-compose:
 dbg: mdbx-dbg
 	$(GO_DBG_BUILD) -o $(GOBIN)/ ./cmd/...
 
-geth:
+geth: mdbx
```

Comment:
Is this intentional?

Reply:
Yes, it didn't work. But it'd be better to remove geth command.